app/Bills.py

```diff
@@ -125,10 +125,10 @@ def fully_populate_bill(cls, jsondata, xmldata, bill_num, bill_type):
         intro_date = jsondata['introduced_at']
         intro_date = datetime.strptime(intro_date, '%Y-%m-%d').date()
 
-	bill_q = Bill.query.filter(Bill.bill_type == bill_type) \
-	    .filter(Bill.bill_num == num) \
-	    .filter(Bill.congress == cong)
-	bills = bill_q.all()
+        bill_q = Bill.query.filter(Bill.bill_type == bill_type) \
+            .filter(Bill.bill_num == num) \
+            .filter(Bill.congress == cong)
+        bills = bill_q.all()
```

Comment:
dunno how this even worked before. Lemme test this out on my server.

Reply:
was wondering the same, maybe you have local changes not pushed?

Reply:
Feel free to push your local changes and I'll rebase this PR if need be.

Reply:
Ah, I have inconsistent use of tabs vs spaces, even on my server copy. So your fixes are legit, and I will keep them.